locator/tablets.cc

```diff
@@ -1002,6 +1002,7 @@ void tablet_aware_replication_strategy::process_tablet_options(abstract_replicat
                                                                replication_strategy_params params) {
     if (ars._uses_tablets) {
         _initial_tablets = params.initial_tablets.value_or(0);
+        _tablet_hints = params.tablet_hints;
```

Comment:
Replication strategy is per keyspace, while tablet hints are per table. So this is not the right place to store he hints. They should be obtained from schema_ptr.

Reply:
Isn't the tablet_aware_replication_strategy per table?

Reply:
But that said, it appears like we can indeed retrieve them from the schema

Reply:
replication strategy is instantiated per keyspace. This one extends `per_table_replication_strategy` but it means that each table has its own erm, replication strategy object is still per keyspace.

Reply:
Dropped in v7

Reply:
> replication strategy is instantiated per keyspace. This one extends `per_table_replication_strategy` but it means that each table has its own erm, replication strategy object is still per keyspace.

I see.  It makes sense, though the name is a bit misleading.  I hope its documented